spec/helpers/registration_helper_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe RegistrationHelper do
+  it 'does something' do
```

Comment:
Yup.  I complained about there not being even a placeholder for the stuff being untested.  So I added that.